examples/data-fetch/pages/index.js

```diff
@@ -2,21 +2,21 @@ import React from 'react'
 import Link from 'next/link'
 import fetch from 'isomorphic-unfetch'
 
-export default class Index extends React.Component {
-  static async getInitialProps () {
-    const res = await fetch('https://api.github.com/repos/zeit/next.js')
-    const json = await res.json()
-    return { stars: json.stargazers_count }
-  }
+const Index = props => {
```

Comment:
Components should use functional syntax:

`function Index(props) {`

Reply:
@timneutkens Thank you Tim, I change it.